### DISCUSSION TITLE
rewrite error

### DISCUSSION THREAD

**Discussion by star-starry-sea** [vercel/community#3588] [Help] [answered]
### Page to Investigate
local debug

### Steps to Reproduce

vercel.json: 
```
{
  "rewrites": [
    {
      "source": "/:path*",
      "destination": "/index.html"
    }
  ]
}
```
I use it in my vue project.
When I run "vercel dev", I got "Unable to find a webpage corresponding to the following URL: http://localhost:3000/" in chrome. 
I run "vercel" to upload it. and visit the preview url, It worked well. And When I removed it, it worked well locally. Why???

**Comment by amyegan** [comment]
If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give us a better view of what is going wrong.

**Reply by star-starry-sea** [reply, marked as answer]
This issue has been resolved in the new version(31.2.3) just released by Vercel. Thank you for your attention.
